### PR TITLE
feat: add shortcut for map filter

### PR DESCRIPTION
**Pull request by nlssakokorhonen**

### CODE REVIEW

File: src/quality_result_gui/quality_error_manager.py [hunk 1]
```diff
@@ -157,6 +162,41 @@ def _add_predefined_filters(self) -> None:
...
+        self,
+        action: QAction,
+        shortcut: str,
+    ) -> None:
+        iface.mainWindow().addAction(action)
+
+        QgsGui.shortcutsManager().registerAction(action, shortcut)
+        action.setShortcut(QKeySequence(shortcut))
+
+    def _toggle_show_errors_on_map_filter(self) -> None:
```
Comment (LKajan) [inline src/quality_result_gui/quality_error_manager.py:187]: Checkboxes have a toggle method already. I think you could connect self.toggle_show_errors_on_map_action.triggered to that directly.

File: src/quality_result_gui/quality_error_manager.py [hunk 2]
```diff
@@ -62,6 +64,8 @@ class QualityResultManager(QObject):
     error_checked = pyqtSignal(str, bool)
     quality_error_selected = pyqtSignal(QualityError, SelectionType)
 
+    SHORTCUT_TOGGLE_ERRORS_ON_MAP_FILTER = "Ctrl+Alt+L"
```
Comment (LKajan) [inline src/quality_result_gui/quality_error_manager.py:67]: I think that `L` stands for Laatu. Should we have the default shortcut in this public repo something more. And that is also reserved shortcut by default. We could change the shortcut to our own in our private implementation.

File: src/quality_result_gui/quality_error_manager.py [hunk 3]
```diff
@@ -157,6 +162,41 @@ def _add_predefined_filters(self) -> None:
             self._attribute_filter.update_filter_from_errors
         )
 
+    def _add_actions(self) -> None:
```
Comment (LKajan) [inline src/quality_result_gui/quality_error_manager.py:165]: Would it work if we just create a simple QShortcut and connect that to checbox's toggle method? Something like:
```python
    def _add_shortcuts(self) -> None:
        self.shortcut = QShortcut(QualityResultManager.SHORTCUT_TOGGLE_ERRORS_ON_MAP_FILTER)
        self.shortcut.setObjectName(tr("Toggle show errors on map filter"))
        self.shortcut.activated.connect(self.dock_widget.show_errors_on_map_check_box.toggle)

        QgsGui.shortcutsManager().registerShortcut(self.shortcut)
```

**Review by LKajan — COMMENTED** [review]
```python
    def _add_shortcuts(self) -> None:
        self.shortcut = QShortcut(QualityResultManager.SHORTCUT_TOGGLE_ERRORS_ON_MAP_FILTER)
        self.shortcut.setObjectName(tr("Toggle show errors on map filter"))
        self.shortcut.activated.connect(self.dock_widget.show_errors_on_map_check_box.toggle)

        QgsGui.shortcutsManager().registerShortcut(self.shortcut)
```

File: src/quality_result_gui/quality_error_manager.py (hunk 1 above)
Comment (rnousia) [reply src/quality_result_gui/quality_error_manager.py]: Changed to use directly checkbox's toggle method

File: src/quality_result_gui/quality_error_manager.py (hunk 2 above)
Comment (rnousia) [reply src/quality_result_gui/quality_error_manager.py]: Changed to Alt+Q (as in quality)

File: src/quality_result_gui/quality_error_manager.py (hunk 3 above)
Comment (rnousia) [reply src/quality_result_gui/quality_error_manager.py]: Implemented as suggested, action removed

**Review by LKajan — APPROVED** [review]

**Pull request merged**